.gitignore

```diff
@@ -1,8 +1,19 @@
+.gitignore
```

Comment:
Also an Eclipse IDE Pattern: by default (I think this can be changed somewhere) Eclipse creates a `.gitignore` file into each Java projects.
Because the root `.gitignore` File is already added to the Git Repo, any change to this file will be added to the "staged modifications" without having to "force add" the file. On the contrary, if you want to add `jxlint-impl/.gitignore` to the "staged modifications" (what you do not want), you need to do `git add -f`.

See also [git still shows files as modified after adding to .gitignore](https://stackoverflow.com/questions/9750606/git-still-shows-files-as-modified-after-adding-to-gitignore).